### PR TITLE
[regressionrange] Use `regressed_by` instead of `cf_has_regression_range`

### PR DESCRIPTION
**Pull request by julietowah**
the "regressed_by" field is now being used as a factor in the categorization of regression ranges
 Fixes: #3759 

Train on Taskcluster: regressionrange

### CODE REVIEW

File: test_suite_analysis/metadata.json
Comment (suhaibmujahid) [inline test_suite_analysis/metadata.json:1]: Unrelated change.

File: bugbug/models/regressionrange.py [hunk 1]
```diff
@@ -80,6 +80,11 @@ def get_labels(self):
                     classes[bug_id] = 1
                 elif bug_data["cf_has_regression_range"] == "no":
                     classes[bug_id] = 0
+            # Check the regressed_by field
```
Comment (suhaibmujahid) [inline bugbug/models/regressionrange.py:83]: Comments in general should describe "why" not "what" the code is doing. I do not think the comment here is necessary.

File: bugbug/models/regressionrange.py [hunk 2]
```diff
@@ -80,6 +80,11 @@ def get_labels(self):
...
+            elif "regressed_by" in bug_data and bug_data["regressed_by"]:
```
Comment (suhaibmujahid) [inline bugbug/models/regressionrange.py:84]: The `cf_has_regression_range` has been removed. We can rely solely on `regressed_by` instead.

**Review by suhaibmujahid — COMMENTED** [review]
Thank you @julietowah for the PR.

Please edit the PR title to be more descriptive. Also, the linking in the description is not pointing to an issue.

File: bugbug/models/regressionrange.py (hunk 1 above)
Comment (julietowah) [reply bugbug/models/regressionrange.py:83]: thank you so much on the feed back i will give the update soon

File: test_suite_analysis/metadata.json
Comment (julietowah) [reply test_suite_analysis/metadata.json:1]: thank you so much 
i have updated the pr title

**Comment by julietowah** [comment]
@suhaibmujahid i love the swift response i get from you and the corrections
i really appreciate
i did not join firefox on time but i i was able to find my way around to some extent because the community was so supportive 
thank you

File: bugbug/models/regressionrange.py [hunk 3]
```diff
@@ -74,12 +74,11 @@ def get_labels(self):
 
             bug_id = int(bug_data["id"])
             if "regressionwindow-wanted" in bug_data["keywords"]:
+                classes[bug_id] = 1
+            elif "regressed_by" in bug_data and bug_data["regressed_by"]:
```
Comment (suhaibmujahid) [inline bugbug/models/regressionrange.py:78]: This could be similified to use only one `if` statement.

File: test_suite_analysis/metadata.json
Comment (suhaibmujahid) [reply test_suite_analysis/metadata.json:1]: Why did you add the `test_suite_analysis/metadata.json` file here? It does not seem related to #3759.

File: bugbug/models/regressionrange.py (hunk 3 above)
Comment (julietowah) [reply bugbug/models/regressionrange.py]: done

File: test_suite_analysis/metadata.json
Comment (julietowah) [reply test_suite_analysis/metadata.json:1]: @suhaibmujahid please can you explain more i don't understand

Comment (julietowah) [reply test_suite_analysis/metadata.json:1]: fixed it

**Review by suhaibmujahid — DISMISSED** [review]
Thank you, @julietowah! LGTM!

**Review by suhaibmujahid — APPROVED** [review]
